### PR TITLE
Fix code block  failing to align left side of lines in Bootstrap 4

### PR DESCRIPTION
**Pull request by atusy**
With Bootstrap 4, first line of code block has extra padding on the left.
I figured out it is because of missing 

```css
pre code {padding: 0}
```

in Bootstrap 4, which exists in Bootstrap 3.

image

## Reprex

````
---
title: "Untitled"
output:
  html_document:
    theme:
      version: 4
---


```
foo
bar
```
````

### CODE REVIEW

**Comment by cderv** [comment]
@cpsievert is this something that should be added into **bslib** directly as this is specific to bootstrap 4 ?

**Comment by atusy** [comment]
Ah yeah that is the good point! Thanks a lot!

**Comment by atusy** [comment]
@cpsievert Thanks. I close this PR!

**Comment by yihui** [comment]
Sorry I didn't understand it. Do you mean this PR will go to **bslib**?

**Comment by atusy** [comment]
I thought @cpsievert 's comment means it.
I am not yet familiar with the structure of **bslib** so I want @cpsievert to handle this issue there.

Oh, but there is still a possibility to merge this and revert it later if **bslib** is coming soon on CRAN without fixing this issue.

**Comment by yihui** [comment]
Let's keep this PR open then. Since it is a fairly simple change, I'm okay with merging and reverting it later when it's fixed upstream.

**Comment by cpsievert** [comment]
FWIW, this has been addressed upstream by https://github.com/rstudio/bslib/commit/dc8c0b5c829be6ec0483a0e89732cc9d84366ee2, but it might make sense to just merge this so we can release bslib after rmarkdown

File: NEWS.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ rmarkdown 2.7
 
 - `knitr_options_pdf()` will now throw a warning when `fig_crop = TRUE` but is disabled because required tools `pdfcrop` and/or `ghostscript` are missing (thanks, @netique, #2016).
 
+- Fix alignment of lines in code blocks generated by `html_document` with Bootstrap 4 theme (thanks, @atusy, #2019).
```
Comment (yihui) [inline NEWS.md:22]:
```suggestion
- Eliminated the unnecessary padding in code blocks in the `html_document` output with Bootstrap 4 themes (thanks, @atusy, #2019).
```

**Review by yihui — APPROVED** [review]
Okay. I'll merge this one for now. Thanks!

**Pull request merged**